src/pages/home/ClipboardCard.tsx

```diff
@@ -16,12 +24,23 @@ export const ClipboardCard: FC<ClipboardCardProps> = ({
   clipboard,
   clipboardId,
 }) => {
-  const { checklist, name } = clipboard ?? {}
+  const { name } = clipboard ?? {}
+  const onButtonClick = () => console.log('Delete clipboard')
+
   return (
-    <div>
-      {name && <div>{name}</div>}
-      <div>ID: {clipboardId}</div>
-      <div>{checklist && <div>Checklist: {checklist.length}</div>}</div>
+    <div css={cardDefault}>
+      <Link css={cardEmptyLink} to={`/clipboard/${clipboardId}`} />
```

Comment:
Doing what airbnb does with there cards :) So we don't end up with nested `<button /> or <a />` which could cause issues.  Setting a relative position for the delete button  so it always show above the `cardEmptyLink`